dashboard/test/ui/features/step_definitions/pd.rb

```diff
@@ -161,7 +161,27 @@
   user.permission = UserPermission::WORKSHOP_ADMIN
 end
 
-And(/^I create some fake applications of each type and status$/) do
+And (/^I create some fake teacher applications$/) do
+  require_rails_env
+  time_start = Time.now
+  application_count = 0
+
+  %w(csd csp).each do |course|
+    Pd::Application::ActiveApplicationModels::TEACHER_APPLICATION_CLASS.statuses.each do |status|
+      application =
+        FactoryGirl.create(
+          Pd::Application::ActiveApplicationModels::TEACHER_APPLICATION_FACTORY,
+          course: course
+        )
+      application.update(status: status)
+      application_count += 1
+    end
+  end
+
+  puts "Created #{application_count} applications in #{Time.now - time_start} seconds."
```

Comment:
I don't see this log printed out on my local machine, and couldn't find it in Saucelabs log (maybe I'm looking at the wrong place). I can take it out if it is useless.